src/mysql_k8s_helpers.py

```diff
@@ -299,9 +301,13 @@ def create_custom_config_file(
 
         if innodb_buffer_pool_chunk_size:
             content.append(f"innodb_buffer_pool_chunk_size = {innodb_buffer_pool_chunk_size}")
-        content.append("")
+
+        # TODO: Enable after GR is enabled
```

Comment:
Remove TODO before merging?

Reply:
strangely enough I've removed and pushed, but the commit is not showing on gh

Reply:
33cbf57